exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -0,0 +1,635 @@
+#pragma once
+
+#include <openssl/ssl.h>
```

Comment:
missing all standard includes for things used in the file. even though the includes are implicitly included now does not mean it will be in the future so please add explicit includes for all used standard library features, such as string_view, utility, optional, any ...

Reply:
Added includes.